README.md

```diff
@@ -40,6 +40,10 @@ HLS stream, code against the regular HTML5 video APIs, and create a
 fast, high-quality video experience across all the big web device
 categories.
 
+Check out the [full documentation](docs/) for details on how HLS works
```

Comment:
We should make an index page of some sort for docs but my main point with this PR was to expand on the adaptive switching info.